clang/test/dpct/helper_files_ref/include/device.hpp

```diff
@@ -59,7 +59,7 @@ class device_info {
   // get interface
   const char *get_name() const { return _name; }
   char *get_name() { return _name; }
-  template <typename WorkItemSizesTy = sycl::id<3>,
+  template <typename WorkItemSizesTy = int *,
```

Comment:
This change will break the compatibility of dpct helper functions.
I advise that this header file device.hpp is not touched, change the dpct migration process.
For example:
Migrate 'int *maxGridSize_ptr = deviceProp.maxGridSize;' 
to 
'int *maxGridSize_ptr = deviceProp.get_max_nd_range_size<int*>();'

Reply:
This fix has been implemented within dpct now.  device.hpp is not changed.